### PR TITLE
Add Turing Network (prefix 51, Same with OAK)

### PR DESCRIPTION
**Pull request by imstar15**

### CODE REVIEW

File: ss58-registry.json [hunk 1]
```diff
@@ -460,11 +460,20 @@
...
       "website": "https://composable.finance"
     },
+    {
+      "prefix": 51,
+      "network": "turing",
+      "displayName": "Turing Network",
+      "symbols": ["TUR"],
+      "decimals": [10, 10],
+      "standardAccount": "*25519",
+      "website": "https://oak.tech/turing/home/"
```
Comment (chrisli30) [inline ss58-registry.json]: Let’s use this one for website, "https://oak.tech/turing/token/"

File: ss58-registry.json [hunk 2]
```diff
@@ -460,11 +460,20 @@
...
+    },
     {
       "prefix": 51,
       "network": "oak",
       "displayName": "OAK Network",
-      "symbols": ["OAK", "TUR"],
+      "symbols": ["OAK"],
```
Comment (chrisli30) [inline ss58-registry.json:476]: Hmm so the previous value ["OAK", "TUR"] doesn’t work for subkey? How is this symbols array translated to subkey? Maybe we should just make subkey accept two values in the array.

File: ss58-registry.json [hunk 3]
```diff
@@ -460,11 +460,20 @@
...
       "decimals": [10, 10],
```
Comment (chrisli30) [inline ss58-registry.json]: You need to change this to [10] too ...

File: ss58-registry.json [hunk 4]
```diff
@@ -460,11 +460,20 @@
       "standardAccount": "*25519",
       "website": "https://composable.finance"
     },
+    {
+      "prefix": 51,
+      "network": "turing",
+      "displayName": "Turing Network",
+      "symbols": ["TUR"],
+      "decimals": [10, 10],
```
Comment (chrisli30) [inline ss58-registry.json]: You need to change this to [10] too ...

**Review by chrisli30 — CHANGES_REQUESTED** [review]

File: ss58-registry.json (hunk 2 above)
Comment (imstar15) [reply ss58-registry.json:476]: Yes, It doesn't work for subkey.

In fact, only the prefix field is read from the ss-58 module in the subkey code, and other fields are not read.

File: ss58-registry.json (hunk 1 above)
Comment (imstar15) [reply ss58-registry.json]: Fixed

File: ss58-registry.json (hunk 3 above)
Comment (imstar15) [reply ss58-registry.json]: Good catch!

File: ss58-registry.json (hunk 4 above)
Comment (imstar15) [reply ss58-registry.json]: Good catch!

**Review by chrisli30 — APPROVED** [review]
LGTM

**Pull request merged**